### PR TITLE
CMR-7237

### PR DESCRIPTION
**Pull request by littleidiot40**
Update CCMEO endpoint to non-whitelisted version.

### CODE REVIEW

File: spec/views/granules/descriptor_document_spec.rb [hunk 1]
```diff
@@ -153,22 +153,30 @@
...
+    expect(rendered).to include("template=\"https://eodms-sgdot.nrcan-rncan.gc.ca/apps/cgi/opensearch/eodms_opensearch_r1.sh?q=&amp;id={geo:id?}&amp;bbox={geo:box?}&amp;dtstart={time:start?}&amp;dtend={time:end?}&amp;pw={startPage?}&amp;startIndex={startIndex?}&amp;count={count?}\">")
   end
   it "is possible to create a collection-specific granule open search descriptor document with a whitespace client id" do
     stub_client_id = stub_model(ClientId)
     stub_client_id.clientId = ' '
     assign(:client_id_model, stub_client_id)
 
     render
-    expect(rendered).to include("template=\"http://ceocat.ccrs.nrcan.gc.ca/cgi-bin/opensearch_r1.sh?q=&amp;id={geo:id?}&amp;bbox={geo:box?}&amp;dtstart={time:start?}&amp;dtend={time:end?}&amp;pw={startPage?}&amp;startIndex={startIndex?}&amp;count={count?}\">")
+    expect(rendered).to include("template=\"https://eodms-sgdot.nrcan-rncan.gc.ca/apps/cgi/opensearch/eodms_opensearch_r1.sh?q=&amp;id={geo:id?}&amp;bbox={geo:box?}&amp;dtstart={time:start?}&amp;dtend={time:end?}&amp;pw={startPage?}&amp;startIndex={startIndex?}&amp;count={count?}\">")
```
Comment (jceaser) [inline spec/views/granules/descriptor_document_spec.rb:171]: do you mean to send nothing to q?

File: spec/views/granules/descriptor_document_spec.rb [hunk 2]
```diff
@@ -153,22 +153,30 @@
...
+  end
+  it "creates a collection-specific granule open search descriptor document with a temporal range limit of 14 days" do
+    stub_client_id = stub_model(ClientId)
+    stub_client_id.clientId = 'foo'
+    assign(:client_id_model, stub_client_id)
+
+    render
+    expect(rendered).to include("<param:Parameter name=\"dtend\" value=\"{time:end}\" minimum=\"0\"  minInclusive = '' maxInclusive = '' maxPeriod=\"P14D\" relativeTo=\"{time:start}\" title=\"Temporal End\" />")
```
Comment (jceaser) [inline spec/views/granules/descriptor_document_spec.rb:179]: nitpick: space before = with minInclusive and maxInclusive.

File: spec/views/granules/descriptor_document_spec.rb [hunk 3]
```diff
@@ -196,6 +204,14 @@
     render
     expect(rendered).to include("template=\"https://eoportal.eumetsat.int/eopos?pi=&amp;pw={startPage?}&amp;si={startIndex?}&amp;c={count?}&amp;bbox={geo:box?}&amp;dtstart={time:start?}&amp;dtend={time:end}&amp;iqd={eop:productQualityStatus?}\">")
   end
+  it "creates a collection-specific granule open search descriptor document with a temporal range limit of 30 days" do
+    stub_client_id = stub_model(ClientId)
+    stub_client_id.clientId = 'foo'
+    assign(:client_id_model, stub_client_id)
+
+    render
+    expect(rendered).to include("<param:Parameter name=\"dtend\" value=\"{time:end}\" minimum=\"0\"  minInclusive = '' maxInclusive = '' maxPeriod=\"P30D\" relativeTo=\"{time:start}\" title=\"Temporal End\" />")
```
Comment (jceaser) [inline spec/views/granules/descriptor_document_spec.rb:213]: nitpick: same formatting as above, space around =

**Review by jceaser — CHANGES_REQUESTED** [review]

File: spec/views/granules/descriptor_document_spec.rb (hunk 1 above)
Comment (littleidiot40) [reply spec/views/granules/descriptor_document_spec.rb:171]: If no dataset id is assigned (as per original test), yes.

File: spec/views/granules/descriptor_document_spec.rb (hunk 2 above)
Comment (littleidiot40) [reply spec/views/granules/descriptor_document_spec.rb]: Updated

**Review by ygliuvt — APPROVED** [review]

File: spec/views/granules/descriptor_document_spec.rb (hunk 1 above)
Comment (jceaser) [reply spec/views/granules/descriptor_document_spec.rb:171]: and the parameter is required I guess?

**Review by jceaser — APPROVED** [review]

**Comment by littleidiot40** [comment]
Manually committed.

**Pull request closed without merging**